data_steward/deid/config/ids/config.json

```diff
@@ -201,6 +201,22 @@
         "date": "DATE_SUB( CAST(:FIELD AS DATE), INTERVAL (:SHIFT) DAY) AS :FIELD",
         "datetime": "TIMESTAMP_SUB( CAST(:FIELD AS TIMESTAMP), INTERVAL (:SHIFT) DAY) AS :FIELD"
     },
+    {
+         "_id": "compute",
+         "id": ["(SELECT research_id FROM :idataset._deid_map WHERE _deid_map.person_id = :value_field) as :FIELD"],
+         "response_id": [
+             "(SELECT ",
+             "CASE WHEN :value_field IS NULL THEN NULL ",
+             "ELSE (",
+                 "SELECT dqrm.research_response_id ",
+                 "FROM :idataset._deid_questionnaire_response_map AS dqrm ",
+                 "WHERE dqrm.questionnaire_response_id = :value_field) ",
+             "END ) AS :FIELD"
+         ],
```

Comment:
I see what happened with the previous patch.  Lines 207-215 needed to be removed, but the entire dictionary was removed instead.  If you can remove these lines, this will be good to merge.

Reply:
Ah, I see. I will remove those and repush!